### DISCUSSION TITLE
Is it possible to download Planetscale and use it locally?

### DISCUSSION THREAD

**Discussion by NetOpWibby** [planetscale/discussion#93] [Q&A] [answered]
From a quick glance at the site, everything you do requires authenticating with the CLI. What if I don't have Internet access? Seems weird to me to have an "always on" database in development.

**Comment by jgreet** [comment, marked as answer]
PlanetScale is built on Vitess managed MySQL 8.0 instances, so you could use MySQL 8 as your offline dev database. That's what we do.

That said we see people making use of online dev branches on PlanetScale all the time. [Online dev environments](https://github.com/features/codespaces) seemed weird at first too.
